stix_shifter_modules/azure_log_analytics/stix_translation/json/stix_2_1/to_stix_map.json

```diff
@@ -0,0 +1,368 @@
+            "key": "x-oca-event.code",
+            "object": "event"
+        }
+    ],
+    "IpAddress": [
+        {
+            "key": "x-ibm-finding.src_ip_ref",
+            "object": "finding"
+        },
+        {
+            "key": "ipv4-addr.value",
+            "object": "ipv4-addr"
+        },
+        {
+            "key": "x-oca-event.ip_refs",
+            "object": "event"
+        }
+    ],
+    "ClientIPAddress": [
+        {
+            "key": "x-ibm-finding.dst_ip_ref",
+            "object": "finding"
+        },
+        {
+            "key": "x-oca-event.ip_refs.clientIp",
+            "object": "event"
+        }
+    ],
+    "StartTime": {
+        "key": "x-ibm-finding.start",
+        "object": "finding"
+    },
+    "EndTime": {
+        "key": "x-ibm-finding.end",
+        "object": "finding"
+    },
+    "ProcessingEndTime": {
+        "key": "x-ibm-finding.end",
+        "object": "finding"
+    },
+    "FirstActivityTime": {
+        "key": "x-ibm-finding.start",
+        "object": "finding"
+    },
+    "FirstModifiedTime": {
+        "key": "x-ibm-finding.start",
+        "object": "finding"
+    },
+    "ClosedTime": {
+        "key": "x-ibm-finding.end",
+        "object": "finding"
+    },
+    "LastActivityTime": {
+        "key": "x-ibm-finding.end",
+        "object": "finding"
+    },
+    "LastModifiedTime": {
+        "key": "x-ibm-finding.end",
+        "object": "finding"
+    },
+    "CreatedTime": {
+        "key": "x-oca-event.created",
+        "object": "event"
+    },
+    "Account": {
+        "key": "x-oca-event.provider",
+        "object": "event"
+    },
+    "QuarantineHelpURL": [
+        {
+            "key": "x-oca-event.url_ref",
+            "object": "event"
+        },
+        {
+            "key": "url.name",
+            "object": "url"
+        }
+    ],
+    "IncidentUrl": [
+        {
+            "key": "x-oca-event.url_ref",
+            "object": "event"
+        },
+        {
+            "key": "url.name",
+            "object": "url"
+        }
+    ],
+    "ProcessName": [
+        {
+            "key": "process.name",
+            "object": "process"
+        },
+        {
+            "key": "x-oca-event.process_ref",
+            "object": "event"
+        }
+    ],
+    "CommandLine": {
+        "key": "process.command_line",
+        "object": "process"
+    },
+    "ProcessId": {
+        "key": "process.pid",
+        "object": "process"
+    },
+    "ParentProcessName": [
+        {
+            "key": "process.parent_ref",
+            "object": "process"
+        },
+        {
+            "key": "x-oca-event.parent_process_ref",
+            "object": "event"
+        }
+    ],
+    "FilePath": [
+        {
+            "key": "file.path",
+            "object": "file"
+        },
+        {
+            "key": "x-oca-event.file_ref.path",
+            "object": "file"
+        }
+    ],
+    "FileHash": [
+        {
+            "key": "file.hashes",
```

Comment:
Needs to be mapped to specific hash type ie. `file.hashes.'MD5'`

Reply:
it needs some post processing on the datasource fields. basically needs to check the hash type and create new datasource fields based on the hash value. The newly created datasource fields can be mapped in to_stix.